yt_dlp/extractor/rcti.py

```diff
@@ -0,0 +1,243 @@
+        'info_dict': {
+            'id': 'v_ex9438',
+            'title': 'md5:2ede828c0f8bde249e0912be150314ca',
+            'display_id': 'md5:62b8d4e9ff096db527a1ad797e8a9933',
+            'description': 'md5:2ede828c0f8bde249e0912be150314ca',
+            'ext': 'mp4',
+            'duration': 93,
+            'timestamp': 1587561540,
+            'upload_date': '20200422',
+            'series': 'iNews Malam',
+            'channel': 'INews',
+        },
+        'params': {
+            'format': 'bestvideo, bestaudio',
+        }
+    }]
+
+    def _search_auth_key(self, webpage):
+        try:
+            self._AUTH_KEY = self._search_regex(
+                r'\'Authorization\':"(?P<auth>[^"]+)"', webpage, 'auth-key')
+        except RegexNotFoundError:
+            pass
+
+    def _real_extract(self, url):
+        video_type, video_id, display_id = re.match(self._VALID_URL, url).groups()
+        webpage = self._download_webpage(url, display_id)
+        self._search_auth_key(webpage)
+
+        video_json = self._call_api(
+            'https://api.rctiplus.com/api/v1/%s/%s/url?appierid=.1' % (video_type, video_id), display_id, 'Downloading video URL JSON')[0]
+        video_url = video_json['url']
+        if 'akamaized' in video_url:
+            # Akamai's CDN requires a session to at least be made via Conviva's API
+            # TODO: Reverse-engineer Conviva's heartbeat code to avoid phantomJS
+            phantom = None
+            try:
+                phantom = PhantomJSwrapper(self)
+                phantom.get(url, webpage, display_id, note2='Initiating video session')
+            except ExtractorError:
+                self.report_warning('PhantomJS is highly recommended for this video, as it might load incredibly slowly otherwise.'
+                                    'You can also try opening the page in this device\'s browser first')
+
+        video_meta, meta_paths = self._call_api(
+            'https://api.rctiplus.com/api/v1/%s/%s' % (video_type, video_id), display_id, 'Downloading video metadata')
+
+        thumbnails, image_path = [], meta_paths.get('image_path', 'https://rstatic.akamaized.net/media/')
+        if video_meta.get('portrait_image'):
+            thumbnails.append({
+                'id': 'portrait_image',
+                'url': image_path + '2000' + video_meta['portrait_image']
+            })
+        if video_meta.get('landscape_image'):
+            thumbnails.append({
+                'id': 'landscape_image',
+                'url': image_path + '2000' + video_meta['landscape_image']
+            })
+
+        formats = self._extract_m3u8_formats(video_url, display_id, 'mp4', headers={'Referer': 'https://www.rctiplus.com/'})
+        self._sort_formats(formats)
+
+        for f in formats:
+            if 'akamaized' in f['url']:
+                f.setdefault('http_headers', {})['Referer'] = 'https://www.rctiplus.com/'  # Referer header is required for akamai CDNs
+
```

Comment:
```suggestion
        for f in formats:
            if 'akamaized' in f['url']:
                f.setdefault('http_headers', {})['Referer'] = 'https://www.rctiplus.com/'

        self._sort_formats(formats)
```

Don't edit `formats` after sorting